src/test/java/com/google/api/codegen/testdata/java_smoke_test_library.baseline

```diff
@@ -31,18 +36,34 @@ public class LibraryServiceSmokeTest {
   public static void main(String args[]) {
     Logger.getLogger("").setLevel(Level.WARNING);
     try {
-      executeNoCatch(args);
-      System.out.println("ok");
+      Options options = new Options();
+      options.addOption("h", "help", false, "show usage");
+      options.addOption(
+          Option.builder()
+              .longOpt("project_id")
+              .desc("Project id")
+              .hasArg()
+              .argName("PROJECT-ID")
+              .required(true)
```

Comment:
Do we want this to be required? Or can it be optional, with a default? Especially in the case that nothing in the smoke test config contains $PROJECT_ID, it is odd for it to be required.